app/server/controllers/UserController.js

```diff
@@ -706,6 +706,49 @@ UserController.checkOutById = function(id, user, callback){
   callback);
 };
 
+/**
+ * [ADMIN ONLY]
+ *
+ * Make user an admin
+ * @param  {String}   userId   User id of the user being made admin
+ * @param  {String}   user     User making this person admin
+ * @param  {Function} callback args(err, user)
+ */
+UserController.makeAdminById = function(id, user, callback){
```

Comment:
Ahh I see -- @revalo, if it's not too difficult, can you go ahead and save the user in this function? Seems like it could be important, and if we don't do it now, we'll never get around to it.